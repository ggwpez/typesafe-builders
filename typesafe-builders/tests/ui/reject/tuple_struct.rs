use typesafe_builders::prelude::*;

#[derive(Builder)]
pub struct Struct(pub u8);

fn main() {
}
