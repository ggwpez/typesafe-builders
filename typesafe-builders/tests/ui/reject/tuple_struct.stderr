error: Builder does not work on unnamed fields
 --> tests/ui/reject/tuple_struct.rs:4:19
  |
4 | pub struct Struct(pub u8);
  |                   ^^^^^^
